### ISSUE TITLE
Can not able to connect to SAS Viya to submit job

### ISSUE THREAD

**Issue by venkattoluchuri** [sassoftware/vscode-sas-extension#8]
I have a client id and client password for authentication. I used to get auth code and access token. i have been using the same in VS Code editor settings for SAS Viya URL , client id , client password and token file saved on my mac desktop.

Attached screen shot

Below is the error message
<img width="1152" alt="vscode" src="https://user-images.githubusercontent.com/19385401/172064815-eba95dda-5a1b-4e6b-92d1-efd79a4f575e.png">

{"service":"compute","detail":{"status":0,"statusText":

**Comment by scnwwu** [comment]
The error message you posted looks incomplete. Is it all you got?
Would you try update the extension to latest version?
Please notice that as long as a token file specified, the extension will authenticate with the access token. The client ID/Secret is not used in this case.

**Comment by venkattoluchuri** [comment]
Re attached error message again.  SAS Execution version is installed v0.0.5.
<img width="783" alt="vs" src="https://user-images.githubusercontent.com/19385401/172107664-04e7ea6d-8a5d-47c0-8dd3-0c069c3f0b2b.png">

**Comment by scnwwu** [comment]
Okay... it looks like a network issue. Your connection was reset.
Are you able to make an API call from command line (for example https://developer.sas.com/apis/rest/#making-an-api-call)?

**Comment by venkattoluchuri** [comment]
Yes. API call is working with the access token. Can we have a quick call please, If you are available ?

**Comment by venkattoluchuri** [comment]
Issue is not resolved. Still had the issue. i was asking a call if possible to resolve over the team's channel

**Comment by venkattoluchuri** [comment]
can we reopen this please, as still issue persist?
